Sources/EpoxyCore/SwiftUI/LayoutUtilities/SizingContainer.swift

```diff
@@ -0,0 +1,77 @@
+// Created by Bryn Bodayle on 1/24/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import SwiftUI
+
+// MARK: - SizingContainer
+
+/// A container which reads the proposed SwiftUI layout and passes it a `SizingContext`. This stores
+/// ideal size as state in order to configure the view layout as the ideal size is updated via the
+/// `SizingContext`.
+public struct SizingContainer<Content: View>: View {
```

Comment:
Do you think it makes sense to make this a bit more specific now that it's part of the public API? I worry that having it in the global namespace along with layout groups helpers may make it ambiguous that this is meant for SwiftUI/Epoxy interop
```suggestion
public struct IdealSizeContainer<Content: View>: View {
```

Reply:
collaborated on some options over slack since this view also can be used for just reading the proposed size from the SwiftUI layout